### PR TITLE
Add a composite index for flutter repo and creation date on Checklist.

### PR DESCRIPTION
**Pull request by yusufm**
Adding a composite index on Checklist entity for FlutterRepositoryPath and CreationDate, as a precursor to enabling display of different repo data on flutter dashboard.

### CODE REVIEW

File: app_dart/index.yaml [hunk 1]
```diff
@@ -13,6 +13,11 @@ indexes:
   - name: Branch
   - name: CreateTimestamp
     direction: desc
+- kind: Checklist
```
Comment (yusufm) [inline app_dart/index.yaml:16]: @CaseyHillers based on the prospective queries we will want to make, should we be adding the repo info to Task entities directly, versus using the property in Checklist?

Comment (CaseyHillers) [reply app_dart/index.yaml:16]: It isn't necessary for get-status. It grabs tasks if they have the parent key of the checklist - https://cs.opensource.google/flutter/cocoon/+/869becfbe169ce10c561f507c8984e2233e1be51:app_dart/lib/src/service/datastore.dart;l=124

File: app_dart/index.yaml [hunk 2]
```diff
@@ -13,6 +13,11 @@ indexes:
...
+  properties:
+  - name: FlutterRepositoryPath
+  - name: CreateTimestamp
```
Comment (CaseyHillers) [inline app_dart/index.yaml:19]: Can we add branch?

Comment (yusufm) [reply app_dart/index.yaml:20]: done

**Review by CaseyHillers — APPROVED** [review]
LGTM

**Pull request merged**